section2/exercises/making_decisions.rb

```diff
@@ -0,0 +1,58 @@
+puts "You enter a dark room with three doors.  Do you go through door #1 or door #2 or door #3?"
```

Comment:
Nice work here. I especially want to call out your use of style and syntax that meets Ruby conventions - referring to indentation, use of spaces and whitespace, etc - it's all spot on, making your code very readable.